src/Contao/View/Contao2BackendView/ActionHandler/AbstractListShowAllHandler.php

```diff
@@ -480,15 +480,20 @@ private function getTableHead(EnvironmentInterface $environment)
         $sorting    = ViewHelpers::getCurrentSorting($environment);
         $columns    = $this->getSortingColumns($sorting);
         foreach ($formatter->getPropertyNames() as $field) {
-            // Skip unknown properties. This may happen if the property is not defined for editing but only listing.
-            if (!$properties->hasProperty($field)) {
-                continue;
+            if ($properties->hasProperty($field)) {
```

Comment:
I would rephrase this to keep the original implementation below but switching only the `'content'` part.
Like:
```php
$tableHead[] = [
    'class'   => 'tl_folder_tlist col_' . $field . (\in_array($field, $columns) ? ' ordered_by' : ''),
    'content' => $properties->hasProperty($field)) 
        ? $properties->getProperty($field)->getLabel()
        : $this->translate($definition->getName() . '.' . $field . '.0', 'contao_' . $definition->getName())
];
```
IMO the implementation is clearer then.